sdk/core/core-http/src/policies/proxyPolicy.ts

```diff
@@ -31,33 +31,33 @@ function loadEnvironmentProxyValue(): string | undefined {
 }
 
 // Check whether the given `uri` matches the noProxyList. If it matches, any request sent to that same `uri` won't set the proxy settings.
-function isBypassed(uri: string) {
+function isBypassed(uri: string): any {
```

Comment:
this should return `boolean` right? Or maybe `boolean | undefined` And the map itself should be typed as `Map<string, boolean>` I think?

Reply:
Done.